manager/gui/src/components/configuration/LrNodeConfiguration.vue

```diff
@@ -121,12 +122,12 @@
                   color="primary"
                   @click="applyNetworkInfo"
                   :disabled="!isNetworkDetailsValid"
-                  :dark="isFormValid"
+                  :dark="isNetworkDetailsValid"
                   >Apply
                 </v-btn>
                 <v-btn
                   color="csmdisabled"
-                  @click="resetNetworkData"
+                  @click="setNetworkInfoAllNodes(true)"
```

Comment:
you would need a confirmation modal before resetting the data to default

Reply:
A confirmation modal has been added.